Re-home "running a dbt project" (#240)
fishtown-analytics/docs.getdbt.com: website/docs/dbt-cli/cli-overview.md

---
title: CLI overview

---

> dbt ships with a Command Line Interface (CLI) for running your dbt project. This way of running dbt a dbt project is free and open source.

Reviewer: ```suggestion
dbt ships with a Command Line Interface (CLI) for running your dbt project. The dbt CLI is free to use and [open source](https://github.com/fishtown-analytics/dbt).
```
